modules/test/playwright/tests/export-import-web/pages/CompanyExportImportPage.ts

```diff
@@ -23,7 +23,12 @@ export class CompanyExportImportPage {
 
 	async export(
 		itemLabel: string,
-		includePermissions: boolean = false
+		includePermissions: boolean = false,
+		endDate: string = '',
+		endTime: string = '',
+		rangeLast: string = '',
+		startDate: string = '',
+		startTime: string = ''
 	): Promise<string> {
```

Comment:
With the approach I'm suggesting, this would now be an optional object with optional strings inside:
```
async export(
		itemLabel: string,
		includePermissions: boolean = false,
		dateOptions?: {
			endDate?: string,
			endTime?: string,
			rangeLast?: string,
			startDate?: string,
			startTime?: string
		}
	): Promise<string>
```
This could be defined as a type itself in a `.d.ts` file inside the types folder also for more clearness
```
{
	endDate?: string,
	endTime?: string,
	rangeLast?: string,
	startDate?: string,
	startTime?: string
}
```